### ISSUE TITLE
Question regarding warnings encountered with Unity (non-CN) version 2020 and above

### ISSUE THREAD

**Issue by johnaqalc** [Unity-Technologies/com.unity.uiwidgets#349]
We've encountered the following warnings when running an android build on Unity 2020.3.36f1 with UIWidgets version 1.5.4_stable:

1.  `Shader warning in 'UIWidgets/canvas_convexFill_cb': use of potentially uninitialized variable (shader_gradient_colorize) at <project-folder>/Packages/UIWidgets-release_1.5.4_stable/Runtime/Resources/shaders/UIWidgets_canvas.cginc(72) (on d3d11)`
    -  Aside from `UIWidgets/canvas_convexFill_cb`, we are also encountering this warning in the following shaders: 
        - `UIWidgets/canvas_strokeAlpha_cb`
        - `UIWidgets/canvas_tex_cb`
        - `UIWidgets/canvas_convexFill`
        - `UIWidgets/canvas_tex`
        - `UIWidgets/canvas_stroke0_cb`
        - `UIWidgets/canvas_stroke0`
        - `UIWidgets/canvas_strokeAlpha`
        - `UIWidgets/canvas_fill1`
        - `UIWidgets/canvas_fill1_cb`

2. `UnobservedTaskException:System.NullReferenceException: Object reference not set to an instance of an object`

External Tools:
- JDK: Installed with Unity
- Android SDK: 30.0.3
- NDK: Installed with Unity
- Gradle: 6.5.1

We'd just like to confirm if these warnings are related to the fact that there is currently no official support for Unity (non-CN) version 2020 and above.

**Comment by zhuxingwei** [comment]
Yes, since uiwidget 1.0 is no longer developed for new version Unity (e.g., 2020 LTS), there might be some incompatible issues between them.
